src/common/Logger.ts

```diff
@@ -1,14 +1,20 @@
 import { ILogEvent } from "../models/ILogEvent";
 import { JWT_MESSAGE } from "../models/enums";
 import { ILogError } from "../models/ILogError";
+import { HttpStatus } from "@dvsa/cvs-microservice-common/api/http-status-codes";
+import { APIGatewayTokenAuthorizerEvent } from "aws-lambda";
 
-export const writeLogMessage = (log: ILogEvent, error?: any) => {
+export const writeLogMessage = (event: APIGatewayTokenAuthorizerEvent, log: ILogEvent, error?: any) => {
   if (!error) {
-    log.statusCode = 200;
+    log.statusCode = HttpStatus.OK;
     console.log(log);
   } else {
     const logError: ILogError = {};
-    log.statusCode = 401;
+    log.statusCode = HttpStatus.UNAUTHORIZED;
+
+    // If the DEBUG_MODE env var is set to true, log the token - only applicable when errors occur
+    log.token = process.env.DEBUG_MODE === "true" ? event.authorizationToken : undefined;
```

Comment:
I think that logging out a bearer token, even using the env var or in any case is a bad idea, and can be a security risk and not only that; they will contain live user names and emails which is sensitive data but if someone gets hold of a recent token, they'll be able to potentially access **all** of the endpoints depending on the roles on the token. What is it specifically from the token that you're after?